### ISSUE TITLE
Debug SVG visualisation output is mis-scaled

### ISSUE THREAD

**Issue by jeffbl** [Shared-Reality-Lab/IMAGE-server#919]
Test extension, targeting unicorn
Turn on Developer mode in options

Go to [this photo](https://images.unsplash.com/photo-1494569934110-6b5e3af685cd?w=500&auto=format&fit=crop)
Get response
Open up Depth Map Visualisation
Open up Semantic Segmentation Visualisation

Note: Overlays looks like they are mis-scaled and do not cover entire underlying photograph:

![image](https://github.com/user-attachments/assets/2dc6bb87-9781-4614-be4d-145b2455a28e)

Expect: Overlays align correctly with underlying graphics

Ping @VenissaCarolQuadros in case any recent SVG work is impacting this

Putting in user issue partially since developer mode is publicly available, but also because we use SVGs for end-user functionality now, so in case this impacts actual experience generation, want to get it sorted out.

**Comment by VenissaCarolQuadros** [comment]
I don't believe the recent SVG changes could have any impact since they were limited to two handlers 'photo-tactile-svg' and 'map-tactile-svg' which are only used by the Monarch (and now the TAT). Nothing besides this was changed in #916.

**Comment by jeffbl** [comment]
Well, shoot. I cannot reproduce this. Is there a plausible fix that went int that would impact this? Otherwise I fear there is some other step that we're missing that causes this. I was seeing it pretty regularly before, so hopefully I'll see it again (if it isn't actually fixed), so we can trace it down...
